docs/en/observability/aws-firehose-index.asciidoc

```diff
@@ -0,0 +1,9 @@
+:doctype: book
```

Comment:
@tommyers-elastic Here's an index file to get you started if you want to get these docs published while I'm away. You can just add this alias and reload your bash profile (or equivalent):

The command for building this locally looks like:

`$GIT_HOME/docs/build_docs --doc $GIT_HOME/observability-docs/docs/en/observability/aws-firehose-index.asciidoc --chunk 2`